backend/migrations/versions/5e5c84138af7_backfill_confidentiality.py

```diff
@@ -74,8 +76,8 @@ def upgrade():
         print('Updating datasets...')
         datasets: [Dataset] = session.query(Dataset).all()
         for dataset in datasets:
-            if dataset.confidentiality not in ['Unclassified', 'Official', 'Secret']:
-                dataset.confidentiality = 'Unclassified'
+            if dataset.confidentiality not in ConfidentialityClassification:
```

Comment:
are you sure this is working?
you could do something like:

if not any(value == dataset.confidentiality for value in ConfidentialityClassification.__dict__.values()):